build/CopyFilesToStagingDir.ps1

```diff
@@ -37,6 +37,7 @@ PublishFile $OverrideDir\DynamicDependency-Override.json $FullPublishDir\
 PublishFile $OverrideDir\PushNotifications-Override.json $FullPublishDir\
 
 PublishFile $FullBuildOutput\WindowsAppRuntime_DLL\Microsoft.WindowsAppRuntime.dll $FullPublishDir\Microsoft.WindowsAppRuntime\
+PublishFile $FullBuildOutput\WindowsAppRuntime_DLL\Microsoft.WindowsAppRuntime.pdb $FullPublishDir\Microsoft.WindowsAppRuntime\
```

Comment:
PDBs weren't in here before? Why now? Omitting them == not-compliant with the Transport Package spec?

Reply:
Not sure. I think they might have just been missed.

I believe the Aggregator does not copy these pdbs to the final WinAppSDK either hence it wasn't necessary.

Reply:
I've double checked the transport package schema and yes they've include pdbs.